src/kirby/components/tabs/tab-button/tab-button.component.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../../scss/utils';
+
+:host {
+  height: 100%;
+  flex: 1 1 0%;
+  max-width: 168px;
+
+  @include media(">=large") {
+    flex: none;
+    padding: 0 size('xs');
+  }
+}
+
+.tab-button {
+  font-size: font-size('xxs');
+
+  .badge {
```

Comment:
BadgeComponent supports css custom variables for `left`, `top`, `position` an `z-index`, so consider loosing the additional wrapper and style the badge through these. See Avatar for an example. Same thing could be achieved for the icon.